radio/src/targets/common/arm/stm32/telemetry_driver.cpp

```diff
@@ -179,6 +205,7 @@ void telemetryPortInvertedInit(uint32_t baudrate)
     case 115200:
       bitLength = 17;
       probeTimeFromStartBit = 23; //because pin is not probed immediately
+//      probeTimeFromStartBit = 25; // TODO: Tarnis used 25, how to merge?
```

Comment:
This timing basically depends on interrupt latency until this code is reached. I believe this has been tuned by hand by either @MikeBland or @bsongis. If I have to pick, I'd leave 23, as the latency went up, not down, in general.